tests/workers_round_robin_bench.ts

```diff
@@ -0,0 +1,72 @@
+const data = "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n";
+const workerCount = 4;
+const cmdsPerWorker = 400;
+
+export interface ResolvableMethods<T> {
+  resolve: (value?: T | PromiseLike<T>) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  reject: (reason?: any) => void;
+}
+
+export type Resolvable<T> = Promise<T> & ResolvableMethods<T>;
```

Comment:
can this not be imported from elsewhere?

Reply:
It only exists within `//js/util.ts`. Maybe std needs some tools for implementing async messaging channels like this.